README.md

```diff
@@ -96,6 +96,18 @@ destroy` or turn off all services with `lando poweroff`.
      - Or run services separately as shown in [[https://github.com/pulibrary/figgy/blob/master/Procfile]]
    - Access Figgy at http://localhost:3000/
 
+### Parallel Tests
+
+If you'd like to run the test suite in parallel do the following:
+
+1. `bundle exec rake servers:start`
+1. `PARALLEL_TEST_FIRST_IS_1=true RAILS_ENV=test rake parallel:setup`
+1. `./bin/turbo_test`
+
+To run in parallel with working coverage, but worse output, do:
```

Comment:
What does "worse output" mean? Output looks as expected to me, but I have test format set in a global .gitconfig file. Actually it's the other one that doesn't respect that format setting.

Reply:
Is it not showing the full test run time all together?